terraform/redis-enterprise.tf

```diff
@@ -0,0 +1,105 @@
+#
+# Provision for GCP MP subscription
+#
+# Ensure the following variables in your shell environment:
+# export GOOGLE_APPLICATION_CREDENTIALS=<your Google APIs credentials>
+# export REDISCLOUD_ACCESS_KEY=<your Redis Enterrpise Cloud api access key>
+# export REDISCLOUD_SECRET_KEY=<your Redis Enterprise Cloud secret key>
```

Comment:
**Question:** How and where are these variables used (`REDISCLOUD_ACCESS_KEY` and `REDISCLOUD_SECRET_KEY`)? Does Terraform (under the hood) look for it?